### DISCUSSION TITLE
How to correctly set InIfBoundary and other parameters

### DISCUSSION THREAD

**Discussion by ivanovuri** [akvorado/akvorado#624] [Q&A]
Hi

The default filter `InIfBoundary = external`.
On the main page, Last flow stands `InIfBoundary	internal`. Looks like it binds somehow automagically ;-).
My setup is very straightforward - I simply turn on sampling on my tagged external facing interfaces.

    unit 1347 {
        description "*** Uplink Transit 1 ***";
        vlan-id 1347;
        family inet {
            sampling {
                input;
                output;
            }
            address 174.104.3.3/31;
        }
    }

P.S. If you can, please drop a few lines about other parameters. Empty or default parameters. How to configure ExporterGroup?
It's empty in my case, but it would be nice to group exporters by location.
ExporterTenant = acme, but where to change it?

**Comment by vincentbernat** [comment]
This is explained here: https://demo.akvorado.net/docs/configuration#core

**Reply by ivanovuri** [reply]
All should be done via classifiers.
Get it thanks.
